Cloud 175 monitoring split (#123)
percona/percona-xtradb-cluster-operator: install/monitoring.md

The Percona Monitoring and Management (PMM) [provides an excellent solution](https://www.percona.com/doc/percona-xtradb-cluster/LATEST/manual/monitoring.html#using-pmm) to monitor Percona XtraDB Cluster.

### Installing the PMM Server

> Following steps are needed to install the PMM Server to monitor Percona XtraDB Cluster on Kubernetes or OpenShift.

Reviewer: It is needed to mention that -- This step is optional If you already have PMM Server installed (inside or outside of Kubernetes)

> This first section installs the PMM Server to monitor Percona XtraDB Cluster on Kubernetes or OpenShift. The following steps are optional if you already have installed the PMM Server. The PMM Server is available on your network and does not require another installation in Kubernetes.

Reviewer 2: ```suggestion
This first section installs the PMM Server to monitor Percona XtraDB Cluster on Kubernetes or OpenShift. The following steps are optional if you already have installed the PMM Server. The PMM Server available on your network does not require another installation in Kubernetes.
```

   * `credentials.username` should correspond to the `serverUser` key in the `pmm` section of the [deploy/cr.yaml](https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml) file.
   * `credentials.password` should correspond to a value of the `pmmserver` secret key specified in `deploy/secrets.yaml` secrets file. Note that password specified in this example is the default development mode password not intended to be used on production systems.

>    ### Installing the PMM Client

Reviewer 2: ```suggestion
### Installing the PMM Client
```

>    The following steps initiate the PMM client installation:

Reviewer 2: ```suggestion
   The following steps are needed for the PMM client installation:
```

> 1. The PMM client is initiated by updating the ``pmm`` section in the [deploy/cr.yaml](https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml) file.

Reviewer 2: ```suggestion
1. The PMM client installation is initiated by updating the ``pmm`` section in the [deploy/cr.yaml](https://github.com/percona/percona-xtradb-cluster-operator/blob/master/deploy/cr.yaml) file.
```
